File: client/capability-token/capability-token.js
var http = require('http'),
    twilio = require('twilio');

http.createServer(function (req, res) {
  // put your Twilio API credentials here
  var accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
  var authToken = "your_auth_token";

  // put your Twilio Application Sid here
  var appSid = "APXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";

  var capability = new twilio.Capability(accountSid, authToken);
  capability.allowClientOutgoing(appSid);
  capability.allowClientIncoming("jenny");
  var token = capability.generate();

  res.end(token);
}).listen(1337, '127.0.0.1');

console.log('Token servin\' server running at http://127.0.0.1:1337/');

File: client/capability-token-outgoing/capability-token.cs
// In Package Manager, run:
// Install-Package Twilio.Client -DependencyVersion HighestMinor

using System.Web.Mvc;
using Twilio;

public class TokenController : Controller
{
  // GET: Token
  public ActionResult Index()
  {
    // put your Twilio API credentials here
    string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    string authToken = "your_auth_token";

    // put your Twilio Application Sid here
    string appSid = "APXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";

    var capability = new TwilioCapability(accountSid, authToken);
    capability.AllowClientOutgoing(appSid);
    string token = capability.GenerateToken();

    return Content(token, "application/jwt");
  }
}
